### DISCUSSION TITLE
MoGe: Accurate Monocular Geometry Estimation

### DISCUSSION THREAD

**Discussion by Billynom8** [nagadomi/nunif#259] [Ideas]
https://wangrc.site/MoGePage/
Is this something that can be added to IW3?

**Comment by nagadomi** [comment]
I plan to support it in the future, but from reading the issues it is too early now.

**Comment by gituser123456789000** [comment]
I made posts about Google Deepmind apparently making a depth map model or something similar to this MoGe.. not sure where the topics wound up, I was using mobile, so no idea what happened...

Anyway:
https://mega-sam.github.io/

Seems accurate but very low resolution for some reason

Edit: I found the topic I started in my history and I suppose I accidentally downvoted my own post.. and posts with (up arrow 0) don't show up.. now I made it 1 and it's showing up again. So that's fixed
